google/cloud/forseti/services/scanner/dao.py

```diff
@@ -47,13 +47,15 @@ class Violation(base):
         __tablename__ = violations_tablename
 
         id = Column(Integer, primary_key=True)
-        model_handle = Column(String(256))
+        inventory_index_id = Column(String(256))
```

Comment:
Maybe we can change it to a more generic name like source_id.

Reply:
So, from our discussions with joe, no change here.  We will keep this specific column for `inventory_index_index`, and add more columns if we have other id's to store.